### PR TITLE
Add dateStyle/timeStyle options for JavaScript DateTimeFormat

### PR DESCRIPTION
**Pull request by romulocintra**
A checklist to help your pull request get merged faster:
- [x] Summarize your changes

Added Compat-data for timeStyle DateStyle 
- [x] Data: link to resources that verify support information (such as browser's docs, changelogs, source control, bug trackers, and tests)
https://www.chromestatus.com/features/5091631933947904
- [x] Data: if you tested something, describe how you tested with details like browser and version
- [x] Review the results of the linter and fix problems reported (If you need help, please ask in a comment!)
- [ ] Link to related issues or pull requests, if any

### CODE REVIEW

File: javascript/builtins/Intl.json [hunk 1]
```diff
@@ -414,6 +414,56 @@
...
+                  "version_added": false
+                },
+                "firefox_android": {
+                  "version_added": false
+                },
+                "ie": {
+                  "version_added": false
+                },
+                "nodejs": {
+                  "version_added": null
```
Comment (vinyldarkscratch) [inline javascript/builtins/Intl.json]: NodeJS is powered by V8, Chrome's own JavaScript engine.  While I'm not really sure what NodeJS version it's in (and we're not really focusing on NodeJS right now), I think setting it to `true` would be better!

```suggestion
                  "version_added": true
```

File: javascript/builtins/Intl.json [hunk 2]
```diff
@@ -414,6 +414,56 @@
...
+                },
+                "opera": {
+                  "version_added": null
```
Comment (vinyldarkscratch) [inline javascript/builtins/Intl.json]:
```suggestion
                  "version_added": "63"
```

File: javascript/builtins/Intl.json [hunk 3]
```diff
@@ -414,6 +414,56 @@
...
+                },
+                "opera_android": {
+                  "version_added": null
```
Comment (vinyldarkscratch) [inline javascript/builtins/Intl.json]: There's no Opera Android out with Chrome "76", so we can set this to `false`.

```suggestion
                  "version_added": false
```

File: javascript/builtins/Intl.json [hunk 4]
```diff
@@ -930,6 +980,56 @@
...
+                  "version_added": false
+                },
+                "firefox_android": {
+                  "version_added": false
+                },
+                "ie": {
+                  "version_added": false
+                },
+                "nodejs": {
+                  "version_added": null
```
Comment (vinyldarkscratch) [inline javascript/builtins/Intl.json]:
```suggestion
                  "version_added": true
```

File: javascript/builtins/Intl.json [hunk 5]
```diff
@@ -930,6 +980,56 @@
...
+                },
+                "opera": {
+                  "version_added": null
```
Comment (vinyldarkscratch) [inline javascript/builtins/Intl.json]:
```suggestion
                  "version_added": "63"
```

File: javascript/builtins/Intl.json [hunk 6]
```diff
@@ -930,6 +980,56 @@
...
+                },
+                "opera_android": {
+                  "version_added": null
```
Comment (vinyldarkscratch) [inline javascript/builtins/Intl.json]:
```suggestion
                  "version_added": false
```

**Comment by romulocintra** [comment]
> Hey there, thanks for your PR, and welcome to BCD! This is looking good to me, just a couple of quick questions/suggestions:
> 
> 1. IE/Edge and Firefox were set to `false`, may I ask where you got this data from?  (Manual testing is a totally valid answer of course, just want to double-check!)

**Well this still in Stage3 and only browser that shipped is Chrome , i did manual testing to and based on the meeting notes from ECMA402 nobody shipped elsewhere**

Thanks for your review i will update fields :)

**Comment by romulocintra** [comment]
Just rebased and pushed changes @vinyldarkscratch thanks for your review let me know if all is ok for you

**Comment by vinyldarkscratch** [comment]
Oh you don't need to force-push and squash your commits in this repository by the way!  We strictly squash merge PRs here in BCD.  :wink:

**Pull request merged**